code/modules/movespeed/modifiers/mobs.dm

```diff
@@ -161,3 +161,7 @@
 
 /datum/movespeed_modifier/goliath_mount
 	multiplicative_slowdown = -26
+
+/datum/movespeed_modifier/settler
+	multiplicative_slowdown = 0.5
```

Comment:
(Bikeshedding) 0.5 seems a little high, it's the same as being the second lowest level of sanity. Maybe 0.2 or 0.25 instead?

Reply:
Sure, seems fine